src/ledger-bridge.ts

```diff
@@ -26,14 +26,29 @@ export type LedgerSignTypedDataResponse = Awaited<
   ReturnType<LedgerHwAppEth['signEIP712HashedMessage']>
 >;
 
+export type LedgerBridgeOptions = Record<string, string | number | object>;
```

Comment:
I'm thinking if this type is enough to cover the possible options that a bridge would need. What if a bridge needs a value like an array of bytes? 

Though this is not the case for now, so this is probably better as is - but maybe would make more sense a `Record<string, unknown>` or, if we need it to be serializable, we can pull the `Json` type from `@metamask/utils`

Reply:
@legobeat what do you think of using unknown type at that case?